src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -83,6 +89,15 @@ private static void setupLogger(Application application) {
 		application.addComponentMenuElement(Logger.class, "OFF logging", (ActionEvent e) -> logger.setLevel(Level.OFF));
 	}
 
+	private static void setupComplexCommands(Application application) {
+		application.addComponentMenu(ComplexCommandTest.class, "ComplexCommandTest");
+		ComplexCommand commandsToSquare = ComplexCommandTest.setupSquareFactory();
+		application.addComponentMenuElement(ComplexCommandTest.class, "Square",
+				(ActionEvent e) -> commandsToSquare.execute());
+		ComplexCommand commandsToRhombus = ComplexCommandTest.setupRhombusFactory();
+		application.addComponentMenuElement(ComplexCommandTest.class, "Rhombus",
```

Comment:
Tests should be executed using the current driver (DriverManager->getCurrentDriver()